e2e/specs/create.task.spec.js

```diff
@@ -11,6 +11,8 @@ describe('pivotal tracker project page add task to story', () => {
     before(() => {
         dashboard = SingIn.loginAs(username, password);
         project = dashboard.openProjectById(projectID);
+        //TODO create project from api
+        //TODO create story from api
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS1135)